src/Symfony/Component/HttpClient/RetryableHttpClient.php

```diff
@@ -133,6 +134,13 @@ public function request(string $method, string $url, array $options = []): Respo
         });
     }
 
+    public function reset()
```

Comment:
I'd suggest moving the implementation to `DecoratorTrait`

Reply:
done ✔️